Index: jvm/selfie-runner-kotest/src/commonMain/kotlin/com/diffplug/selfie/kotest/SelfieExtension.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/jvm/selfie-runner-kotest/src/commonMain/kotlin/com/diffplug/selfie/kotest/SelfieExtension.kt b/jvm/selfie-runner-kotest/src/commonMain/kotlin/com/diffplug/selfie/kotest/SelfieExtension.kt
--- a/jvm/selfie-runner-kotest/src/commonMain/kotlin/com/diffplug/selfie/kotest/SelfieExtension.kt	(revision 5ee91c50b7b814b88596b67470c68ab8a273c6c9)
+++ b/jvm/selfie-runner-kotest/src/commonMain/kotlin/com/diffplug/selfie/kotest/SelfieExtension.kt	(date 1770190650932)
@@ -23,7 +23,6 @@
 import io.kotest.core.extensions.TestCaseExtension
 import io.kotest.core.listeners.AfterProjectListener
 import io.kotest.core.listeners.FinalizeSpecListener
-import io.kotest.core.source.SourceRef
 import io.kotest.core.spec.Spec
 import io.kotest.core.test.TestCase
 import io.kotest.engine.test.TestResult
@@ -59,12 +58,7 @@
     }
   }
   private fun snapshotFileFor(testCase: TestCase): SnapshotFileProgress {
-    val classOrFilename: String =
-        when (val source = testCase.source) {
-          is SourceRef.ClassSource -> source.fqn
-          is SourceRef.ClassLineSource -> source.fqn
-          is SourceRef.None -> TODO("Handle SourceRef.None")
-        }
+    val classOrFilename: String = testCase.descriptor.root().id.value
     return system.forClassOrFilename(classOrFilename)
   }
